tests/test_analysis.py

```diff
@@ -633,43 +653,6 @@ def test_idainfo(kernel32_idb, version, bitness, expected):
         assert idainfo.cc_size_ldbl == 8
 
 
-@kern32_test(
-    [
-        if_exists(720, 32, None),
-        if_exists(720, 64, None),
-        if_exists(730, 32, None),
-        if_exists(730, 64, None),
-        if_exists(740, 32, None),
-        if_exists(740, 64, None),
-        if_exists(750, 32, None),
-        if_exists(750, 64, None),
-    ]
-)
-def test_idainfo720plus(kernel32_idb, version, bitness, expected):
```

Comment:
Why remove this?

Reply:
I think the `test_idainfo` is enough, this looks superfluous
https://github.com/williballenthin/python-idb/blob/dbbc2e7126207ef1b6410c37a6ef8bbf6adeb945/tests/test_analysis.py#L616-L653